app/src/main/java/com/jerboa/ui/components/post/composables/PostOptionsDropdown.kt

```diff
@@ -428,104 +432,114 @@ fun PostOptionsDropdown(
-                    }
+            val (removeText, removeIcon) =
+                if (postView.post.removed) {
+                    Pair(stringResource(R.string.restore_post), Icons.Outlined.Restore)
+                } else {
+                    Pair(stringResource(R.string.remove_post), Icons.Outlined.GppBad)
+                }
 
-                PopupMenuItem(
-                    text = removeText,
-                    icon = removeIcon,
-                    onClick = {
-                        onDismissRequest()
-                        onRemoveClick(postView)
-                    },
+            PopupMenuItem(
+                text = removeText,
+                icon = removeIcon,
+                onClick = {
+                    onDismissRequest()
+                    onRemoveClick(postView)
+                },
+            )
+            BanPersonPopupMenuItem(postView.creator, onDismissRequest, onBanPersonClick)
+
+            // Only show ban from community button if its a local community
+            if (postView.community.local) {
+                BanFromCommunityPopupMenuItem(
+                    BanFromCommunityData(
+                        person = postView.creator,
+                        community = postView.community,
+                        banned = postView.creator_banned_from_community,
+                    ),
+                    onDismissRequest,
+                    onBanFromCommunityClick,
                 )
-                BanPersonPopupMenuItem(postView.creator, onDismissRequest, onBanPersonClick)
-
-                // Only show ban from community button if its a local community
-                if (postView.community.local) {
-                    BanFromCommunityPopupMenuItem(
-                        BanFromCommunityData(
-                            person = postView.creator,
-                            community = postView.community,
-                            banned = postView.creator_banned_from_community,
-                        ),
-                        onDismissRequest,
-                        onBanFromCommunityClick,
-                    )
-                }
             }
+        }
 
-            // You can do these actions on mods above you
-            if (amMod || amAdmin) {
-                val (lockText, lockIcon) =
-                    if (postView.post.locked) {
-                        Pair(stringResource(R.string.unlock_post), Icons.Outlined.LockOpen)
-                    } else {
-                        Pair(stringResource(R.string.lock_post), Icons.Outlined.CommentsDisabled)
-                    }
+        // You can do these actions on mods above you
+        if (amMod || amAdmin) {
+            PopupMenuItem(
+                text = stringResource(R.string.view_votes),
+                icon = ImageVector.vectorResource(R.drawable.up_filled),
+                onClick = {
+                    onDismissRequest()
+                    onViewVotesClick(postView.post.id)
```

Comment:
This feature is only available in certain lemmy versions, use the FeatureFlag to enable it. `FF.listAdminVotes()`